bench

```diff
@@ -47,17 +47,28 @@ if [[ -d "$(pwd)/.sass-cache" ]]; then
 	rm -rf "$(pwd)/.sass-cache"
 fi
 
+# Create a directory for flamegraphs
+if [[ -n $FLAMEGRAPH ]]; then
+	mkdir -p "docs/$VERSION"
+fi
+
 for SITE in $(cat "site-list"); do
 	echo ""
 	echo "Sampling: $SITE"
 	echo ""
 	SOURCE="$TMPDIR/source/${SITE##*/}"
 	DESTINATION=${SOURCE/source/destination}
+	SVG_PATH="docs/$VERSION/${SITE##*/}"
 	if [[ ! -d $SOURCE ]]; then
 		git clone --recurse-submodules -q "$SITE" "$SOURCE"
 	fi
 	for ((i=0; i<NUMBER_OF_RUNS; ++i)); do
 		"$TIME" -ao "$RESULTS" -f"$VERSION,%U,$SITE" \
 			bundle exec jekyll build -s "$SOURCE" -d "$DESTINATION" --trace
 	done
+
+	if [[ -n $FLAMEGRAPH ]]; then
+		rbspy record --file "$SVG_PATH" -- bundle exec jekyll build -s "$SOURCE" -d "$DESTINATION"
```

Comment:
You're going to run `jekyll build` 3 times per list-item and then once more with `rbspy` recording..?
Can't it be such that `rbspy` *conditionally* records the third `build` command..?